src/site/content/en/blog/gamepad/index.md

```diff
@@ -0,0 +1,348 @@
+A `Gamepad` has the following fields:
+
+- `id`:
+  An identification string for the gamepad.
+  This string identifies the brand or style of connected gamepad device.
+- `index`:
+  The index of the gamepad in the navigator.
+- `connected`:
+  Indicates whether the gamepad is still connected to the system.
+- `timestamp`: The last time the data for this gamepad was updated.
+- `mapping`: The button and axes mapping in use for this device. Currently the only mapping is `"standard"`.
+- `axes`: An array of values for all axes of the gamepad, linearly normalized to the range of
+  `-1.0`–`1.0`.
+- `buttons`: An array of button states for all buttons of the gamepad.
+
+Note that buttons can be digital (pressed or not pressed) or analog (for example, 78% pressed).
+This is why buttons are reported as `GamepadButton` objects, with the following attributes:
+
+- `pressed`: The pressed state of the button (`true` if the button is currently pressed,
+  and `false` if it is not pressed.
+- `touched`: The touched state of the button. If the button is capable of detecting touch,
+  this property is `true` if the button is currently being touched, and `false` otherwise.
+- `value`: For buttons that have an analog sensor, this property represents the amount by
+  which the button has been pressed, linearly normalized to the range of `0.0`–`1.0`.
+
+One additional thing that you might encounter, depending on your browser and the gamepad you have,
+is a `vibrationActuator` property.
+This field is currently implemented in Chrome and earmarked for
+[merging](https://github.com/w3c/gamepad/pull/68) into the
+[Gamepad Extensions](https://w3c.github.io/gamepad/extensions.html) spec.
+
+The schematic overview below, taken
+[straight from the spec](https://w3c.github.io/gamepad/#fig-visual-representation-of-a-standard-gamepad-layout:~:text=Figure%201%20Visual%20representation%20of%20a%20standard%20gamepad%20layout.),
+shows the mapping and the arrangement of the buttons and axes on a generic gamepad.
+
+<figure class="w-figure">
+  <img src="gamepad.svg"
+       alt="Schematic overview of the button and axes mappings of a common gamepad."
+  >
+  <figcaption class="w-figcaption">
+    Visual representation of a standard gamepad layout
+    (<a href="https://w3c.github.io/gamepad/#fig-visual-representation-of-a-standard-gamepad-layout:~:text=Figure%201%20Visual%20representation%20of%20a%20standard%20gamepad%20layout.">Source</a>).
+  </figcaption>
+</figure>
+
+### Being notified when a gamepad gets connected
+
+To learn when a gamepad is connected, listen for the `gamepadconnected`
+event that triggers on the `window` object.
+When the user connects a gamepad, which can either happen via USB or via Bluetooth,
+a `GamepadEvent` is fired that has the gamepad's details in an aptly named `gamepad` property.
+Below, you can see an example from an Xbox 360 controller that I had lying around
+(yes, I am into retro gaming).
+
+```js/3-12
```

Comment:
I fee like the highlighting is not necessary at this point.

Reply:
Fair remark.